### DISCUSSION TITLE
代理人档案 | 凯撒·金

### DISCUSSION THREAD

**Discussion by justafish8787** [share121/inter-knot#1329] [其他]
「我就是凯撒，卡吕冬之子的老大。」
凯撒·金

「随便坐吧，像自己家一样。」
「既然来了外环，你就是卡吕冬之子的客人！」
「呃…大老爹，她说投桃报李是什么意思啊？」

中文CV——吴秋韵
日文CV——佐藤利奈
![1723354658799](https://github.com/user-attachments/assets/f3bb3e95-b40f-4b81-a9b1-d65d64ff5b9b)
![1723354662202](https://github.com/user-attachments/assets/ec1a9c17-a43d-4520-afb6-ce9c823b0075)
![1723354666537](https://github.com/user-attachments/assets/8f5ad84f-5e34-4a70-af4b-fe0b38913c40)

**Comment by GanlinSama** [comment]
真的是 **_大_** 姐头啊

**Comment by ZhuYuan-148** [comment]
外环的人吗？嗯……我去问问外环的人归不归治安局管
